testnode-scripts/config.ts

```diff
@@ -151,16 +151,16 @@ function writeGethGenesisConfig(argv: any) {
 function writeConfigs(argv: any) {
 	const chainInfoFile = path.join(consts.configpath, "l2_chain_info.json")
     const baseConfig = {
-        "l1": {
+        "parent-chain": {
             "url": argv.l1url,
```

Comment:
This is out of date due to the rpcclient change and will need updated, but I'll leave that to a separate PR